src/index.js

```diff
@@ -294,20 +294,25 @@ window.addEventListener('load', function() {
 
   $('#scanAnother').on('click', () => {
     navigateTo('scan');
-  })
+  });
```

Comment:
in our case it's isn't really necessary but I don't mind 😁 
Some contributors use it, some do not. I personally prefer with 😁